web/front/app.js

```diff
@@ -52,6 +53,13 @@ export default {
   },
 
   watch: {
+    showDrilldownAlert(v) {
+      if (v) {
+        setTimeout(() => {
+          this.showDrilldownAlert = false;
+        }, 1000)
+      }
+    },
     "items.length": function() {
```

Comment:
let's mostly follow https://standardjs.com/ - thus always use single quotes for strings